src/genegraph/server.clj

```diff
@@ -115,4 +117,6 @@
     (if (env/transformer-mode?)
       (run-server-transformer nil)
       (run-server-genegraph nil))
-    (run-migration)))
+    (if (= "snapshot" (first args))
+      (do (snapshot/run-snapshots (rest args)))
+      (run-migration))))
```

Comment:
This might be an opportunity to clean up some of this code, let's discuss how best to handle different approaches to selecting run modes for genegraph.